apps/datahub/src/app/record/record-otherlinks/record-otherlinks.component.ts

```diff
@@ -44,15 +46,23 @@ export class RecordOtherlinksComponent implements AfterViewInit {
 
   constructor(
     public facade: MdViewFacade,
-    private changeDetector: ChangeDetectorRef
+    private changeDetector: ChangeDetectorRef,
+    private breakpointObserver: BreakpointObserver
   ) {}
 
   updateView() {
     this.changeDetector.detectChanges()
   }
 
   ngAfterViewInit() {
-    // this is required to show the pagination correctly
     this.changeDetector.detectChanges()
   }
+
+  isMobile$ = this.breakpointObserver.observe([Breakpoints.XSmall])?.pipe(
```

Comment:
I don't know why you need to detect screen size in js here, you can handle responsiveness only by using tailwind css classes (sm:, md:, ...) this will keep homogeneity regarding media queries breakpoints. (This may need a refacto between the carousel and the block component maybe)